### DISCUSSION TITLE
Anyway to Dockerized ani-cli

### DISCUSSION THREAD

**Discussion by AreebMughal** [pystardust/ani-cli#1465] [Q&A]
Hi, 

I have been trying to install ani-cli in my windows but getting error of repositories and even if it is installed ani-cli stopped running randomly. Although I followed the exact guideline mentioned in Readme file.

I've also tried to dockerized the ani-cli but didn't complete it. 

So, I'm asking if there is dockerized version of ani-cli or any one can help me in this. 

Thank you advance for the help.

**Comment by JDevWebb** [comment]
Wow, literally a year to date, anyway.
Here is a docker-compose file to start a dockerized ani-cli, dockerfile is in the comment below, put both in same directory.
[docker-compose.yml](https://github.com/user-attachments/files/24248614/docker-compose.yml)

Dockerfile:
```
FROM alpine:latest

RUN apk add --no-cache \
    bash \
    curl \
    grep \
    sed \
    coreutils \
    ffmpeg \
    mpv \
    aria2 \
    openssl \
    git \
	fzf

RUN git clone https://github.com/pystardust/ani-cli.git /tmp/ani-cli && \
    cd /tmp/ani-cli && \
    cp ani-cli /usr/local/bin/ && \
    chmod +x /usr/local/bin/ani-cli && \
    rm -rf /tmp/ani-cli

WORKDIR /anime

CMD ["/bin/bash"]
```
